client/app/main/student/partial/outreaches.html

```diff
@@ -39,8 +39,8 @@
         <dt>Notes</dt>
         <dd>
           <div class="input-group">
-            <input type="text" class="form-control" ng-model="outreach.newNote"/>
-            <span class="input-group-btn">
+            <textarea type="text" class="form-control" ng-model="outreach.newNote" cols="30" rows="2" style="resize: vertical;"></textarea>
```

Comment:
I think we should revert this change for now. We aren't supporting multi line notes:

![screen shot 2016-03-16 at 4 25 42 pm](https://cloud.githubusercontent.com/assets/394393/13832012/3f9ac214-eb95-11e5-8b02-d6bd6f6dee16.png)

Using a textarea will give an impression that notes can multiple lines. Then after adding the notes the lines will be concatenated.